apis/src/components/organisms/header.rs

```diff
@@ -1,19 +1,62 @@
 use crate::components::atoms::next_game_button::NextGameButton;
+use crate::components::molecules::chat_and_controls::ChatAndControls;
 use crate::components::organisms::{
     darkmode_toggle::DarkModeToggle,
     dropdowns::{CommunityDropdown, MobileDropdown, UserDropdown},
 };
+use crate::providers::game_state::GameStateSignal;
 use crate::providers::AuthContext;
+use hive_lib::GameControl;
 use leptos::*;
 use leptos_router::use_location;
 use shared_types::TimeMode;
 
 #[derive(Clone)]
 pub struct Redirect(pub RwSignal<String>);
 
+#[derive(Clone)]
+pub struct ControlsSignal {
+    pub hidden: RwSignal<bool>,
+    pub notify: Signal<bool>,
+}
+
+#[derive(Clone, Copy)]
+pub struct MobileChat(pub RwSignal<bool>);
```

Comment:
I think this should be renamed because you use this not only when you are referring to chat things. This is more general whether the page is displayed in the mobile view, right?